src/HotChocolate/ElasticSearch/test/Data.ElasticSearch.Tests/__snapshots__/StringPrefixTests.ElasticSearch_AndField_WithNegation_query.snap

```diff
@@ -6,18 +6,18 @@
           "bool": {
             "must": [
               {
-                "prefix": {
+                "wildcard": {
                   "bar": {
```

Comment:
What is faster?

Reply:
Good question. I am not enough into es to answer this question. I rewrote it to use a querystring anyway. Because otherwise it did not work to search for escaped special characters. 

I guess in a further iteration it would be necessary to enable the user for customization. 

ES has a thousand ways to search the same thing in other scenarios. And it always depends on the way it was indexed. My implementations depend on all standard settings. Means everything is tokenized etc. by the default index settings.

There have to be more attributes or other mechanics to fulfill all scenarions.

Two times I force to use the keyword field. One time for sorting and the other time for startsWith, endsWith and contains.

That might not work for all users.

Reply:
@A360JMaxxgamer  standard  is good for now 👍